### ISSUE TITLE
Bypassing directory specific tests / setup + teardown for Object Store backends

### ISSUE THREAD

**Issue by snafus** [indigo-iam/wlcg-jwt-compliance-tests#45]
Hi, 
  RAL-LCG2 (se-ral-test-xrootd ) storage is Ceph, with librados Object Store backend. As such, it has no concept of a directory. 
  While some attempt is made to 'fake' a posix system (e.g. directory create), other operations (e.g stat / deletions of non-existent directories) can not be faked. 
  
  Our test endpoint is currently failing the compliance tests in a few places; i.e. (https://ci.cloud.cnaf.infn.it/view/wlcg/job/wlcg-jwt-compliance-tests/job/master/lastSuccessfulBuild/artifact/reports/reports/latest/se-ral-test-xrootd/report.html#) 

A couple of the failures appear to be in the teardown step (i.e. deletion of a directory), and I believe we should be able to pass the test itself, except for the teardown component.

The other failure, which is directory related "Create directory not allowed with storage.create scope and partial path", we might need to look at on the storage side. 

In summary, would it be possible (via some configuration flag ?) to bypass sections of tests that rely on a directory structure, where it's not a critical component of the test, e.g. in the directory removal of a teardown step for object store backends. 

Many thanks,
James
